src/components/Game/Game.js

```diff
@@ -124,9 +121,13 @@ class GameView extends Component {
 GameView.propTypes = {
     dispatch: PropTypes.func.isRequired,
     correctWords: PropTypes.array.isRequired,
+    submittedAnswer: PropTypes.string.isRequired,
```

Comment:
perhaps it was me who came up with `submittedAnswer` name in the first place but maybe `submitted` prefix does not convey much meaning so perhaps it would be cleaner to rename to `answer` :thinking: 

but this is open to debate, I don't have very strong opinion here, just something to consider

more general comment: names are really important and there are there are two general **conflicting** rules:
1) a name should describe the value
2) a name should be concise